edx_django_utils/db/read_replica.py

```diff
@@ -0,0 +1,134 @@
+"""
+Tools for making queries read from the read-replica database, rather than from the writer database.
```

Comment:
I personally would find it helpful if you could provide some guidance on when these should be used.  Should you strive to use the read-replica for any querying that can use it?  Is it more important to get long running queries to use it?  Other?